src/sentry/static/sentry/app/views/organizationDashboard/overviewDashboard.jsx

```diff
@@ -20,7 +20,14 @@ class OverviewDashboard extends AsyncView {
   }
 
   renderBody() {
-    return <Dashboard releases={this.state.releases} {...overviewDashboard} />;
+    return (
+      <Dashboard
+        releases={this.state.releases}
+        busy={this.state.loading}
+        {...overviewDashboard}
+        {...this.props}
```

Comment:
Is it just router that we need from here now? If that's the case can we just pass `router={this.props.router}`. It's often difficult for the reader to figure out what's actually needed with {...this.props}

Reply:
Hmm it's needed to override `overviewDashboard` for tests as well :/ I've destructured it explicitly and added a comment: https://github.com/getsentry/sentry/pull/12861/commits/5fd64f8ec598baac219e5c83ae5724aef8e1dea6